src/Microsoft.AspNetCore.Razor.Evolution/TagHelperAttributeDescriptor.cs

```diff
@@ -127,30 +113,8 @@ public string TypeName
         }
 
         /// <summary>
-        /// The <see cref="TagHelperAttributeDesignTimeDescriptor"/> that contains design time information about
-        /// this attribute.
-        /// </summary>
-        public TagHelperAttributeDesignTimeDescriptor DesignTimeDescriptor { get; set; }
-
-        /// <summary>
-        /// Determines whether HTML attribute <paramref name="name"/> matches this
-        /// <see cref="TagHelperAttributeDescriptor"/>.
+        /// A summary of how to use a tag helper.
         /// </summary>
-        /// <param name="name">Name of the HTML attribute to check.</param>
-        /// <returns>
-        /// <c>true</c> if this <see cref="TagHelperAttributeDescriptor"/> matches <paramref name="name"/>.
-        /// <c>false</c> otherwise.
-        /// </returns>
-        public bool IsNameMatch(string name)
-        {
-            if (IsIndexer)
-            {
-                return name.StartsWith(Name, StringComparison.OrdinalIgnoreCase);
-            }
-            else
-            {
-                return string.Equals(name, Name, StringComparison.OrdinalIgnoreCase);
-            }
-        }
+        public string Summary { get; set; }
```

Comment:
Interesting... Hmmm. If we did that then we wouldn't need a `DisplayName`. I think I like this approach.